CI/travis/before_install_darwin

```diff
@@ -2,6 +2,7 @@
 
 . CI/travis/lib.sh
 
+brew unlink python@2
```

Comment:
this looks a bit hacky;

curios: what is the error that this fixes?

Reply:
In the latest build python upgrade from 3.5 to 3.6 fails because python2 is already linked.
Will modify this to install packages, not to install and upgrade them.